src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -661,6 +661,12 @@ The following are literal names and should not be translated: ABI, 'libs/armeabi
     <value>Cannot determine ABI of native library '{0}'. Move this file to a directory with a valid Android ABI name such as 'libs/armeabi-v7a/'.</value>
     <comment>The following are literal names and should not be translated: ABI, 'libs/armeabi-v7a/'
 {0} - The file path</comment>
+  </data>
+  <data name="XA4301_ABI_RID" xml:space="preserve">
+    <value>Cannot determine ABI of native library '{0}'. The RuntimeIdentifier '{1}' does not target Android platforms.</value>
```

Comment:
This message isn't actionable; what should the user *do* to fix this?

For that matter, what's it *mean*?  This reads like two separate bits of information: we can't determine the ABI, a'la "normal" `XA4301_ABI`, in which case the library needs to be moved into a different directory (?), but what's that have to do with `RuntimeIdentifier`?

Reply:
> This message isn't actionable; what should the user do to fix this?

I don't think there is much they can do? I could log the name of the NuGet package it came from, and suggest they remove it?

See details here: https://github.com/dotnet/runtime/pull/45927

Basically, if you add a NuGet package that is missing `android-*` RID for native libraries, the dotnet/sdk will potentially fall back to including `linux-*` native libraries. I tried to get them to change the behavior, but it is setup this way for other reasons.

Reply:
Looking closer at the item metadata available:

![image](https://user-images.githubusercontent.com/840039/102365874-a4022d00-3f7d-11eb-9acd-6a403ff1c708.png)

We don't even have `%(RuntimeIdentifier)`, so I changed the message to be:

```
warning XA4301: Cannot determine ABI of native library 'C:\src\xamarin-android\packages\sqlitepclraw.lib.e_sqlite3.linux\1.1.11\runtimes\linux-arm64\native\libe_sqlite3.so'. Remove the 'SQLitePCLRaw.lib.e_sqlite3.linux' NuGet package, or notify the library author.
```

Let me know if you have other ideas, thanks.